Xamarin.Forms.Core/Menu.cs

```diff
@@ -0,0 +1,95 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+
+namespace Xamarin.Forms
+{
+	public class Menu : BaseMenuItem, IList<Menu>
+	{
+		readonly ObservableCollection<Menu> _menus = new ObservableCollection<Menu>();
+
+		readonly ObservableCollection<MenuItem> _items = new ObservableCollection<MenuItem>();
+
+		public Menu()
+		{
+			_items.CollectionChanged += (s, e) => OnPropertyChanged(nameof(Items));
```

Comment:
I guess this is a temporary hack, but it redraws the entire menu

Reply:
yeah it's true